src/ws_server.cu

```diff
@@ -310,8 +314,13 @@ class AprilTagHandler : public seasocks::WebSocket::Handler {
         frame_counter++;
 
         auto overallstart = std::chrono::high_resolution_clock::now();
-        if (gui_rotate_img_) {
-          rotateImage(&bgr_img, 180.0);
+        //Let's check the time this takes, can always combine to one call if both are true later.
+        //Best case scenario is we don't place the camera wrong so we do not need this method at all.
+        if (flipVertical_) {
+          flipVertical(&bgr_img);
```

Comment:
you changed the signature of the flipVertical method above, so this won't compile.  If you want to change back to your original way of calling this method, I'm fine with that too.   there was nothing inherently wrong with it, it just looks a little strange.

Reply:
yep sorry - noticed this now